tests/lib/rook/base.py

```diff
@@ -81,9 +89,10 @@ def install(self):
 
         logger.info("Wait for OSD prepare to complete "
                     "(this may take a while...)")
-        pattern = re.compile(r'.*rook-ceph-osd-prepare.*worker.*Completed')
+        pattern = re.compile(r'.*rook-ceph-osd-prepare.*Completed')
         common.wait_for_result(
-            self.kubernetes.kubectl, "--namespace rook-ceph get pods",
+            self.kubernetes.kubectl, "--namespace rook-ceph get pods"
+            "-l app=rook-ceph-osd-prepare",
```

Comment:
Need an extra whitespace here as this is coming up as "pods-l".

Reply:
done